QuizApp/User/UserViewController.swift

```diff
@@ -1,27 +1,135 @@
 import Foundation
 import UIKit
+import SnapKit
+import Combine
 
 class UserViewController: UIViewController {
 
+    private var userViewModel: UserViewModel!
+
+    var gradientBg: CAGradientLayer!
+    var usernameTitleLabel: UILabel!
+    var usernameLabel: UILabel!
+    var nameTitleLabel: UILabel!
+    var nameLabel: UILabel!
+    var logOutButton: UIButton!
+
+    private var cancellables = Set<AnyCancellable>()
+
+    init(viewModel: UserViewModel) {
+        super.init(nibName: nil, bundle: nil)
+        self.userViewModel = viewModel
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
-
         buildViews()
     }
 
     func buildViews() {
         createViews()
         styleViews()
         defineLayoutForViews()
+        bind()
+        userViewModel.getUser()
+    }
+
+    @objc func handleLogOut() {
+        userViewModel.logout()
+    }
+
+    func bind() {
+        userViewModel
+            .$username
+            .sink { username in
+                self.usernameLabel.text = username
+            }.store(in: &cancellables)
```

Comment:
Each function call should go into new line.
And check if you have a memory leak here.